startupscript/post-startup.sh

```diff
@@ -54,20 +57,31 @@ exec 2>&1
 
 # The apt package index may not be clean when we run; resynchronize
 apt-get update
-apt install -y jq curl tar wget
+apt install -y jq curl fuse tar wget
 
 # Create the target directories for installing into the HOME directory
 ${RUN_AS_LOGIN_USER} "mkdir -p '${USER_BASH_COMPLETION_DIR}'"
 ${RUN_AS_LOGIN_USER} "mkdir -p '${USER_HOME_LOCAL_SHARE}'"
 
-# As described above, have the ~/.bash_profile source the ~/.bashrc
+
+# Custom application behavior when opening a terminal window will vary.
+#
+# Some application that run in custom environments will by default run
+# an interactive non-login shell, which sources the ~/.bashrc.
+#
+# Others will open a login shell, which sources the ~/.bash_profile.
+#
+# For consistency across these as many environments as possible, this startup
+# script writes to  ~/.bashrc, and has the ~/.bash_profile source the ~/.bashrc
+
 cat << EOF >> "${USER_BASH_PROFILE}"
 
 if [[ -e ~/.bashrc ]]; then
  source ~/.bashrc
 fi
 
 EOF
+chown "${USER_NAME}:${USER_PRIMARY_GROUP}" "${USER_BASH_PROFILE}"
```

Comment:
do you mind fixing it here too? https://github.com/verily-src/workbench-app-devcontainers/blob/master/startupscript/install-java.sh#L16